src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -64,26 +64,33 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = CollectAlignmentSummaryMetrics.USAGE_SUMMARY + CollectAlignmentSummaryMetrics.USAGE_DETAILS,
-        usageShort = CollectAlignmentSummaryMetrics.USAGE_SUMMARY,
+        usage = "Using read outputs from NGS technologies, this tool provides detailed metrics" +
```

Comment:
Let's replace NGS by HTS (for high-throughput sequencing) everywhere. This stuff was "Next-generation" 15 years ago. Now it's just sequencing.